edsnlp/utils/filter.py

```diff
@@ -61,6 +62,10 @@ def filter_spans(
         Whether to return discarded spans.
     label_to_remove : str, optional
         Label to remove. If set, results can contain overlapping spans.
+    sort_key: Callable[Span, Any], optional
```

Comment:
mkdocstrings needs a space before and after the colon to parse the types...

```suggestion
    sort_key : Callable[Span, Any], optional
```